snuba/web/rpc/v1/trace_item_attribute_values.py

```diff
@@ -82,8 +82,8 @@ def _build_query(
     This query will match the first 10000 occurrences of an attribute value and then deduplicate them,
     this gives a large speedup to the query at the cost of ordering and paginating all values
     """
-    if request.limit > 1000:
-        raise BadSnubaRPCRequestException("Limit can be at most 1000")
+    if request.limit > 10000:
+        raise BadSnubaRPCRequestException("Limit can be at most 10000")
```

Comment:
Looking to be able to pass 1001 as the way pagination is done on the product side to request `limit + 1` items and if the extra item is there, we know there's another page.